Core/clim-basic/recording.lisp

```diff
@@ -378,15 +378,13 @@ recording stream. If it is T, *STANDARD-OUTPUT* is used.")
 (defun highlight-output-record-rectangle (record stream state)
   (with-identity-transformation (stream)
     (multiple-value-bind (x1 y1 x2 y2)
-        (output-record-hit-detection-rectangle* record)
+        (bounding-rectangle* record)
```

Comment:
why does this change? is the comment about the hit detection rectangle below bogus?

Reply:
comment is alright (that hit detection rectangle may be bigger), but this function highlights output-record-rectangle, not its hit detection range. Also, this improves performance greatly for bigger output records with lots of repainting.